includes/Core/Modules/Modules.php

```diff
@@ -1131,14 +1131,14 @@ function( Module $module ) {
 	 *
 	 * @since 1.50.0
 	 *
-	 * @return array List of recoverable modules.
+	 * @return Module_With_Owner[] array List of recoverable modules.
 	 */
 	public function get_recoverable_modules() {
```

Comment:
As a public method, this should have added tests for it by convention. Since we don't have a need for it publicly yet, let's make it `protected` for now. This will receive added testing in https://github.com/google/site-kit-wp/issues/4724.